lms/djangoapps/edxnotes/tests.py

```diff
@@ -142,34 +145,42 @@ def setUp(self):
         Setup a dummy course content.
         """
         super(EdxNotesHelpersTest, self).setUp()
-        ClientFactory(name="edx-notes")
-        self.course = CourseFactory.create()
-        self.chapter = ItemFactory.create(category="chapter", parent_location=self.course.location)
-        self.chapter_2 = ItemFactory.create(category="chapter", parent_location=self.course.location)
-        self.sequential = ItemFactory.create(category="sequential", parent_location=self.chapter.location)
-        self.vertical = ItemFactory.create(category="vertical", parent_location=self.sequential.location)
-        self.html_module_1 = ItemFactory.create(category="html", parent_location=self.vertical.location)
-        self.html_module_2 = ItemFactory.create(category="html", parent_location=self.vertical.location)
-        self.vertical_with_container = ItemFactory.create(category='vertical', parent_location=self.sequential.location)
-        self.child_container = ItemFactory.create(
-            category='split_test', parent_location=self.vertical_with_container.location)
-        self.child_vertical = ItemFactory.create(category='vertical', parent_location=self.child_container.location)
-        self.child_html_module = ItemFactory.create(category="html", parent_location=self.child_vertical.location)
-
-        # Read again so that children lists are accurate
-        self.course = self.store.get_item(self.course.location)
-        self.chapter = self.store.get_item(self.chapter.location)
-        self.chapter_2 = self.store.get_item(self.chapter_2.location)
-        self.sequential = self.store.get_item(self.sequential.location)
-        self.vertical = self.store.get_item(self.vertical.location)
-
-        self.vertical_with_container = self.store.get_item(self.vertical_with_container.location)
-        self.child_container = self.store.get_item(self.child_container.location)
-        self.child_vertical = self.store.get_item(self.child_vertical.location)
-        self.child_html_module = self.store.get_item(self.child_html_module.location)
 
-        self.user = UserFactory.create(username="Joe", email="joe@example.com", password="edx")
-        self.client.login(username=self.user.username, password="edx")
+        # There are many tests that are comparing locators as returned from helper methods. When using
```

Comment:
Yeah, but right now the apps (LMS and Studio) can't support it, and we were getting major version conflict issues in studio before we hid the versions.